llvm/utils/lit/tests/Inputs/shtest-umask/lit.cfg

```diff
@@ -0,0 +1,5 @@
+import lit.formats
+
+config.name = "shtest-umask"
+config.suffixes = [".txt"]
+config.test_format = lit.formats.ShTest(execute_external=False)
```

Comment:
You'll need to add this:
```
if sys.platform.startswith("win") or sys.platform.startswith("cygwin"):
    config.available_features.add("system-windows")
```
because this sandboxed configuration doesn't run the parent lit.cfg. But with that, and changing the expectation in shtest-umask.py to `Unsupported`, it should work. (A hacked-up version does work in my Windows environment.)

Reply:
Still needs this suggested change, otherwise good to go.

Reply:
Done. Can you please verify that it works? It's pretty annoying that the bots do not run `check-lit` and I have no access to a Windows box that can build LLVM.

Reply:
Bots not running `check-lit` does seem like an oversight. I will apply your patch and try it out tomorrow.